update_employees.py

```diff
@@ -118,6 +119,15 @@ def drop_empty_positions(records_hr, key="pidm"):
     return [r for r in records_hr if r.get(key)]
 
 
+def get_employee_data_local():
+    with open("data.xml", "r") as fin:
+        records_hr_unfiltered = fin.read()
+        json_raw = wddx.loads(records_hr_unfiltered)
+        #  remove weird leading slashes from data contents
+        json_clean = json_raw[0].replace("//", "")
+        return json.loads(json_clean)
```

Comment:
We lost VPN access to the API, so until that's fixed this hack makes it possible to save a copy of the response payload in a local file so that we can mock the API.